bfdd: Code Readiability Changes

ramapalleti opened the pull request:
Code Readiability Changes

ramapalleti commented:
> Can you explain why you think this makes that function more readable? It seems more obscure to me

Hello Mark, As part of the code review, I identified and removed unused arguments within the function to align with other functions and improve code readability. 

Thanks & Regards
Ramanjaneya

mjstapp commented:
but so if "readability" is the goal ... why decorate the function, why not just update the signature? 

> > Can you explain why you think this makes that function more readable? It seems more obscure to me
> 
> Hello Mark, As part of the code review, I identified and removed unused arguments within the function to align with other functions and improve code readability.
> 
> Thanks & Regards Ramanjaneya

ramapalleti commented:
> but so if "readability" is the goal ... why decorate the function, why not just update the signature? I think the higher-level question is: these kinds of one-line PRs still take some reviewing work, still introduce diffs with existing branches, etc. It's not clear that there's a benefit?
> 
> > > Can you explain why you think this makes that function more readable? It seems more obscure to me
> > 
> > 
> > Hello Mark, As part of the code review, I identified and removed unused arguments within the function to align with other functions and improve code readability.
> > Thanks & Regards Ramanjaneya



> but so if "readability" is the goal ... why decorate the function, why not just update the signature? I think the higher-level question is: these kinds of one-line PRs still take some reviewing work, still introduce diffs with existing branches, etc. It's not clear that there's a benefit?
> 
> > > Can you explain why you think this makes that function more readable? It seems more obscure to me
> > 
> > 
> > Hello Mark, As part of the code review, I identified and removed unused arguments within the function to align with other functions and improve code readability.
> > Thanks & Regards Ramanjaneya

Hello Mark, 
From next PR will address all these issues.  Thank You.

donaldsharp commented:
@mjstapp I think this now addresses your comments?

mjstapp commented:
well, not really: if there's going to be a meaningful change, then let's look at that change and review _it_ . but just churning this one line has really taken way too much time already.

> @mjstapp I think this now addresses your comments?

Jafaral commented:
> From next PR will address all these issues. Thank You.

If that is the case, why do a separate PR  for this anyway? I agree with @mjstapp , this change isn't useful as-is at all.

ramapalleti commented:
> > From next PR will address all these issues. Thank You.
> 
> If that is the case, why do a separate PR for this anyway? I agree with @mjstapp , this change isn't useful as-is at all.

OK. Will close this PR.
